packages/pyright-internal/src/analyzer/backgroundAnalysisProgram.ts

```diff
@@ -98,8 +100,8 @@ export class BackgroundAnalysisProgram {
     }
 
     setFileOpened(filePath: string, version: number | null, contents: string, options: OpenFileOptions) {
-        this._backgroundAnalysis?.setFileOpened(filePath, version, contents, options);
-        this._program.setFileOpened(filePath, version, contents, options);
+        this._backgroundAnalysis?.setFileOpened(filePath, version, [{ text: contents }], options);
```

Comment:
Putting the events back made it easier to track them in the program setFileOpened. We could do something else too if we wanted to keep the setFileOpened simpler.

I was thinking potentially:
- We'd just return full edits for the entire file. This can be slow though.
- We track the edits at a higher level as they come into the service.